config/routes.rb

```diff
@@ -1,4 +1,17 @@
 Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
+  mount Rswag::Api::Engine => '/api-docs'
+
+  namespace :v1 do
+    namespace :me do
+      resources :reservations
+    end
+    resources :users
+    resources :cars do
+      resources :reservations
```

Comment:
only thhe routes that are really needed should be defined in routes.rb. Please correct this